bin/yang_get_stats.py

```diff
@@ -228,13 +235,13 @@ def gather_backups_compilation_stats(self):
                     total_result = str(int(failed_result) + int(passed_with_warning_result) + int(passed_result))
                 extracted_date = self._extract_date_from_filename(filename)
                 if (datetime.date.today() - extracted_date).days > 30:
-                    self.remove_old_html_files.append(path_to_file)
+                    self.remove_old_html_file_paths.append(path_to_file)
                 yang_page_compilation_stats[date2num(extracted_date)] = {
                     'total': total_result,
                     'warning': passed_with_warning_result,
                     'success': passed_result,
                 }
-            if int(args.days) == -1:
+            if int(self.args.days) == -1:
```

Comment:
I would prefer to save this `days` to variable and then access it as `self.days`
Do not need to save `args` as class variable